dev/merge_arrow_pr.py

```diff
@@ -328,12 +229,13 @@ def format_issue_output(issue_type, issue_id, status,
     else:
         components = ', '.join((getattr(x, "name", x) for x in components))
 
-    if issue_type == "jira":
-        url = '/'.join((JIRA_API_BASE, 'browse', issue_id))
-    else:
-        url = (
-            f'https://github.com/{ORG_NAME}/{PROJECT_NAME}/issues/{issue_id}'
-        )
+    url_id = issue_id
+    if "GH" in issue_id:
+        url_id = issue_id.replace("GH-", "")
+
+    url = (
+        f'https://github.com/{ORG_NAME}/{PROJECT_NAME}/issues/{url_id}'
+    )
```

Comment:
Can we simplify this?

```suggestion
    url = f'https://github.com/{ORG_NAME}/{PROJECT_NAME}/issues/{url_id}'
```